storage/rocksdb/ha_rocksdb.cc

```diff
@@ -14066,8 +14066,12 @@ int ha_rocksdb::external_lock(THD *const thd, int lock_type) {
       }
     }
   } else {
-    if (my_core::thd_tx_isolation(thd) < ISO_READ_COMMITTED ||
-        my_core::thd_tx_isolation(thd) > ISO_REPEATABLE_READ) {
+    // Check for unsupported transaction isolation levels but allow them for DD
+    // tables. In the latter case they get silenty clamped to RC or RR, which is
+    // good enough for DD operations.
+    if (table_share->table_category != TABLE_CATEGORY_DICTIONARY &&
```

Comment:
could we check whether table name is DD table?

Reply:
`table_category == TABLE_CATEGORY_DICTIONARY` is a cached result of `dd::get_dictionary()->is_dd_table_name(schema_name, table_name)`, so in that way it is already that check, just cheaper, in fact, if possible, I'd convert other string-based checks to this one in the SE layer, if possible.